### PR TITLE
[ROCM] build brake fix 231129

### PR DESCRIPTION
**Pull request by pemeliya**
This is a build brake fix caused by these new changes to gpu_driver: 

https://github.com/openxla/xla/commit/fede5db53eac81a77e33a70094cf889a8a0810dd
https://github.com/openxla/xla/commit/c58bc738fe27187eae50c8aa7243b8bb19e5d26f

Besides, we have also renamed hipblaslt_plugin to workaround hipblas-lt runtime loading problems for rocm-6.0

@xla-rotation: could you please have a look ?  thanks !

### CODE REVIEW

**Review by ddunl — APPROVED** [review]

File: xla/stream_executor/rocm/rocm_driver_wrapper.h [hunk 1]
```diff
@@ -104,15 +104,21 @@ namespace wrap {
...
   __macro(hipGraphAddChildGraphNode)                \
+  __macro(hipGraphAddMemAllocNode)                  \
   __macro(hipGraphAddMemcpyNode)                    \
   __macro(hipGraphAddMemcpyNode1D)                  \
-  __macro(hipGraphExecChildGraphNodeSetParams)      \
+  __macro(hipGraphAddMemsetNode)                    \
   __macro(hipGraphCreate)                           \
   __macro(hipGraphDebugDotPrint)                    \
   __macro(hipGraphDestroy)                          \
+  __macro(hipGraphExecChildGraphNodeSetParams)      \
```
Comment (ddunl) [inline xla/stream_executor/rocm/rocm_driver_wrapper.h:114]: I think there's extra whitespace on the end of this line

Comment (akuegel) [reply xla/stream_executor/rocm/rocm_driver_wrapper.h:114]: To be clear, this would block the automatic merging, because the linter is complaining.

Comment (pemeliya) [reply xla/stream_executor/rocm/rocm_driver_wrapper.h]: thanks for pointing out! fixed

**Review by akuegel — APPROVED** [review]

**Comment by i-chaochen** [comment]
@xla-rotation wonder when this PR will be merged? Thanks!

**Pull request closed without merging**